### PR TITLE
README: Clarify when the .get() result is available

### PR DESCRIPTION
**Pull request by finnp**
Hey,

I feel like writing the example like this, makes clearer when the result of `writable.get()` will be available. ☺️

PS: Thanks for this great module!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,14 @@ Install:
...
 
 // Read objects from input...
-const writable = input.pipe(sorter(42));
-
-// Returns top 42 sorted entries, sync.
-console.log(writable.get());
-
+const writable = input
+  .pipe(sorter(42))
+  .on('end', () => {
```
Comment (addaleax) [inline README.md]: Shouldn’t this be `finish`? `sorter()` returns a writable stream, so I don’t think `end` works?

**Review by addaleax — APPROVED** [review]
Thanks! I have a question though – I’m not 100 % sure but I think it should be The Other Event?™

File: README.md (hunk 1 above)
Comment (finnp) [reply README.md]: Ah, yes you are right. I'll update the PR!

**Comment by addaleax** [comment]
Thanks, merged!

Fwiw, I think I wrote the original text because if the data is available synchronously on the input stream, the result should also be available synchronously. But I do agree that this is clearer!

**Comment by finnp** [comment]
Yeah, I see how it made sense, but it seemed a bit confusing to me in the beginning, so I thought I'll just send you a PR 😊

**Pull request merged**